text/dotnet-core/rfcs/0000-debug-support.md

```diff
@@ -0,0 +1,79 @@
+# Enable Remote Debug Support
+
+## Proposal
+
+The .NET Core buildpack should enable remote debugging of applications in a
+manner that conforms to [Project
+RFC0037](https://github.com/paketo-buildpacks/rfcs/blob/main/text/0037-remote-debug.md).
+To accomplish this, the buildpack will need to install the Visual Studio
+Debugger into the application run image. The debugger can attach to a running
+`dotnet` process and be bound to a client-side debugger via STDIN across a
+connection invoked via `docker exec` or `kubectl exec`.
+
+## Motivation
+
+Remote debugging is a useful practice for understanding complex program logic
+and interactions in remote environments including production. Providing this
+feature for .NET Core developers will bring this buildpack into alignment with
+the types of features we already provide for developers in other language
+families like Java.
+
+## Implementation
+
+Implementing this feature will require changes in 2 existing buildpacks,
+`dotnet-publish` and `dotnet-execute`, and the addition of a new buildpack,
+`vsdbg`, to the `dotnet-core` language-family buildpack.
+
+### `vsdbg`
```

Comment:
I think it would be good to call out that we will only support Visual Studio and VSCode (?) initially and describe how this could be extended in the future to support other IDEs like [Rider](https://www.jetbrains.com/rider/).